packages/dx-react-scheduler-material-ui/src/templates/drag-drop/resize.jsx

```diff
@@ -81,4 +63,31 @@ ResizeBase.defaultProps = {
   forwardedRef: undefined,
 };
 
-export const Resize = withStyles(styles, { name: 'Resize' })(ResizeBase);
+const StyledResizeBaze = styled(ResizeBase)(({ theme: { spacing } }) => {
```

Comment:
I think the best solution there should be styling the `div` instead styling the `ResizeBase`